evergreen/Dockerfile

```diff
@@ -16,4 +16,5 @@ SHELL [ "poetry", "run" ]
 RUN [ "python3","manage.py","makemigrations" ]
 
 ENTRYPOINT [ "./docker-entrypoint.sh" ]
-CMD [ "python3", "manage.py", "runserver", "0.0.0.0:8080" ]
+#CMD [ "python3", "manage.py", "runserver", "0.0.0.0:8080" ]
+CMD ["daphne", "-b", "0.0.0.0", "-p", "8080", "evergreen.asgi:application"]
```

Comment:
`There can only be one CMD instruction in a Dockerfile or build stage. Only the last one will have an effect. dockerfile-utils(37)`

Reply:
fixed